pkg/plugins/resources/shell/environments.go

```diff
@@ -0,0 +1,70 @@
+package shell
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Environments []Environment
+
+func (e *Environments) ToStringArray() []string {
```

Comment:
```suggestion
func (e *Environments) ToStringSlice() []string {
```

Nit: Arrays in go are fixed length  arrays, here you're returning a slice of string, so this method should be called `ToStringSlice`.

Reply:
That's a very good suggestion, thank you very much